dist/husky.js

```diff
@@ -33403,6 +33463,7 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
                 data: null,
                 instanceName: '',
                 searchInstanceName: null,
+                storageName: null,
```

Comment:
In an ideal world the `instanceName` should also be unique, because we open the door for unwanted side-effects otherwise. However, when thinking about it, it would be quite a big BC break, because all the event names change... So I think we leave it as it is.